Projects/COM/COM04/COM04-AWS200.md

```diff
@@ -0,0 +1,53 @@
+# COM04-AWS200 - Create a cluster of virtual machines using docker swarm
+
+## Cloud Service Provider
+
+- Amazon Web Services
+
+## Difficulty
+
+- Level 200 (Intermediate)
```

Comment:
Anything Docker Swarm related is not 200 level. This is easily 300 if not 400, it requires extensive experience not only in Docker itself but the whole swarm concept. This is material at the level of the AWS Certified DevOps Professional certificate.

For a reference on a 200 level, check https://github.com/100DaysOfCloud/100DaysOfCloudIdeas/blob/master/Projects/SEC/SEC01/SEC01-AWS200.md.

Reply:
Agreed. This project involves writing complicated user data scripts that set up EC2 nodes, Docker[Swarm] knowledge and NGinx or similar web server. Also knowledge on how to prove that the swarm is indeed working. Not sure if 300 or 400 but def. not 200.